src/Cli/dotnet/NugetPackageDownloader/NuGetPackageDownloader.cs

```diff
@@ -1,5 +1,6 @@
 // Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
```

Comment:
nit: I'm guessing you didn't mean to add an extra commented line here

Reply:
👍 That's been bothering me for a while now, especially when creating new files in VS. I think it comes from https://github.com/dotnet/sdk/blob/60feccb2da91f75c7ec6a58ee2f600b27f4a1f85/.editorconfig#L159